src/FieldWarning/Assets/LoadingScreen.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Threading;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class LoadingScreen : MonoBehaviour
+{
+
+    private Slider slider;
+    private Text descLbl;
+
+    static public Queue<Loading> workers = new Queue<Loading>();
+    private Thread _thread;
+
+
+
+
+    // Start is called before the first frame update
+    void Start()
+    {
+        slider = transform.Find("Slider").GetComponent<Slider>();
+        descLbl = transform.Find("LoadingLbl").GetComponent<Text>();
+
+
+    }
+
+
+    void Update()
+    {
+        if (workers.Count > 0) 
+        {
```

Comment:
We write the opening brace inline except for method and class definitions (kernel style). That said I think cuoshen and a lot of other people default to your style, so we could change that rule if you ask cuoshen and he prefers the style used here.